setup.py

```diff
@@ -0,0 +1,38 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import os
+import sys
+from setuptools import setup
+
+try:
+    from testr.setup_helper import cmdclass
+except ImportError:
+    cmdclass = {}
+
+if "--user" not in sys.argv:
+    share_path = os.path.join(sys.prefix, "share", "jobwatch")
+    web_path = os.path.join(sys.prefix, "www", "jobwatch")
```

Comment:
I think there is a problem here because in a clean install of the environment this will create the `www` directory and put files there, but on HEAD the `www` directory needs to be a link to `/proj/sot/ska/www` (which is really `/data/mta4` at this time).

I'd suggest making `overlib.js` be regular package data and then copying to the destination `rootdir` if needed at runtime.

Reply:
I was thinking in a clean install intended to be a production copy, that one makes the `www` directory to start (like I did for the new HEAD ska3/flight) and this works.  In a clean install without `www` this makes it and then doesn't work, but I don't think we care for non-production environments.  In the already-set-up production environment, this should put the file we want in the right place. I'm not seeing the problem.

Could also use a post-install hook for conda, or the runtime copy you suggestion, but didn't seem necessary.

Reply:
Last time I checked, conda won't do a clean install into a directory that exists and has stuff in it. For the shiny production install you did the link *after* the install. Is that different now? 

In any case, we should minimize the external data `data_files` to the extent possible. Basically `overlib.js` is just another output file analogous to, and required by, the generated content. It's worth noting that the template file hardcodes the location of overlib.js at `../overlib.js`, so this will be broken unless the output `rootdir` happens to be a subdirectory of `web_path`. Instead let's do this right and have `overlib.js` live in `rootdir`.

Reply:
I don't know about "clean install into a directory that exists and has stuff in it" in this context.  I had done a test install into a bare ska, and it made the 'www' directory and stuck the overlib.js in it.  Then I removed the package, made the 'www' a link to a temp directory and repeated that test without issue.  I don't think we'd care about deleting a previous www/jobwatch directory as long as we didn't have to mess with `www` itself?

I'm not arguing if you want to minimize data_files. But along those lines... are you actually looking at the overlib.js content?  I confess I've never been to the skawatch index page where that works.

Reply:
The real goal is to make the fresh install process as simple and clean as possible, and have zero post-install steps for particular packages. The only thing we absolutely need now is the `data` link. 

What you seem to be proposing is a post-install step for this package to remove the package-created `www` and replace it with the link, and then do some by-hand copying of `overlib.js` to put it back into the right place. Then you end up having part of the configured Ska3 flight distribution living on /data/mta4. This is really not ideal.

So please just do what I suggested.

Reply:
Sure.  

PS I was not proposing that.  I was mentioning a) it seemed to work without post-install steps (though yes, in a clean install you'd now make 'www' link before installing the ska conda packages instead of after). b) if it needed post-install steps I'd do them as conda post-install.sh c) I still don't think anybody uses the overlib data.